### PR TITLE
New post: Bedrock Resource Packs

### PR DESCRIPTION
**Pull request by KuroiJigoku**

### CODE REVIEW

File: _posts/minecraft/bedrock/2024-07-13-bedrock-rp.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title:  "Resource Packs"
```
Comment (iamdeka) [inline _posts/minecraft/bedrock/2024-07-13-bedrock-rp.md:3]:
```diff
- title:  "Resource Packs"
+ title: "How to add a resource pack to your server"
```

File: _posts/minecraft/bedrock/2024-07-13-bedrock-rp.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+category: 
+    - Bedrock
+    - Getting-started
+tags: Server
+description: "Uploading Resource Packs into your Bedrock Server"
```
Comment (iamdeka) [inline _posts/minecraft/bedrock/2024-07-13-bedrock-rp.md:8]:
```diff
- description: "Uploading Resource Packs into your Java Server"
+ description: "How to make your Bedrock server require a Resource Pack"
```

File: _posts/minecraft/bedrock/2024-07-13-bedrock-rp.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+permalink: /minecraft/bedrock/server/resource-pack
+author: "Kuroi Jigoku"
+toc: false
+---
+
+## Introduction
+
+Resource packs in Minecraft can significantly enhance the visual and auditory experience of the game. For server owners, offering custom resource packs can create a unique environment for players. This guide will walk you through the steps to upload and apply a resource pack to your Bedrock Minecraft server, ensuring all players can enjoy the customized experience you’ve designed. Whether you're adding custom textures, sounds, or user interfaces, follow these instructions to seamlessly integrate your resource pack with your Bedrock server.
```
Comment (iamdeka) [inline _posts/minecraft/bedrock/2024-07-13-bedrock-rp.md:16]: AI again?

File: _posts/minecraft/bedrock/2024-07-13-bedrock-rp.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+1. Download and extract the resource pack. If the file is in `.mcpack` format, rename the file extension to `.zip` format and extract it.
+![Renaming to zip](/assets/images/posts/rename.webp)
+
+2. In the extracted folder, there is a `manifest.json` file. Open it with Notepad or any other Text Editor.
+
+3. Minimize the file for now, we will need it later on.
+
+> In this tutorial, We will use SFTP for transferring files.
+> You may refer to [this tutorial](https://kb.falixnodes.net/falix/dashboard/general/sftp)
```
Comment (iamdeka) [inline _posts/minecraft/bedrock/2024-07-13-bedrock-rp.md:43]:
```diff
- > You may refer to [this tutorial](https://kb.falixnodes.net/falix/dashboard/general/sftp)
+ > You may refer to [this](https://kb.falixnodes.net/falix/dashboard/general/sftp) on how to connect to your server via SFTP
```

File: _posts/minecraft/bedrock/2024-07-13-bedrock-rp.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+
+## Transferring Files
+
+1. Log in to your Minecraft Server's SFTP.
```
Comment (iamdeka) [inline _posts/minecraft/bedrock/2024-07-13-bedrock-rp.md:47]: Perhaps using login is not the best idea, use connect
It also doesn't have to be a Minecraft server. Just say server.

File: _posts/minecraft/bedrock/2024-07-13-bedrock-rp.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+   ![manifest-packid](/assets/images/posts/manifest-packid.webp)
+   ![pack id](/assets/images/posts/BedrockPackID.webp)
+
+9. Replace the `version` value with the **version** numbers from the `manifest.json` file.
+    ![manifest-ver](/assets/images/posts/manifest-ver.webp)
+    ![version](/assets/images/posts/BedrockResourceVer.webp)
+
+10. Save the file.
+
+11. (re)Start your server.
```
Comment (iamdeka) [inline _posts/minecraft/bedrock/2024-07-13-bedrock-rp.md:80]:
```diff
- 11. (re)Start the server
+ 11. Start or restart the server
```

File: _posts/minecraft/bedrock/2024-07-13-bedrock-rp.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+
+{: .success}
+
+> You have Successfully added a Resource pack into your server.
```
Comment (iamdeka) [inline _posts/minecraft/bedrock/2024-07-13-bedrock-rp.md:84]:
```diff
- > You have Successfully added a Resource pack into your server.
+ > You have successfully added a Resource Pack to your server.
```

**Review by iamdeka — CHANGES_REQUESTED** [review]

**Review by Mocab — APPROVED** [review]
If anyone has anything else to add lmk

**Pull request merged**